src/components/export-modal/export-graph.js

```diff
@@ -32,8 +36,10 @@ const exportGraph = ({ format, theme, graphSize, mockFn }) => {
     width = Math.min(width, maxWidth);
     height = Math.min(height, maxWidth * (height / width));
   }
-  clone.setAttribute('width', width);
-  clone.setAttribute('height', height);
+  if (hasGraph) {
+    clone.setAttribute('width', width);
+    clone.setAttribute('height', height);
+  }
```

Comment:
it might be worth to rethink about the `else` scenerio for the empty flowchart - we could perhaps apply a standard `width` and `height` in the meantime for the sake of allowing us to still export and empty graph while we look into the flow when exporting an empty graph?

Reply:
I think it works as is without needing an explict height - I don't think we need to think any more about this extreme edge case. I tidied up the console errors for the sake of completeness as I don't like exposing uncaught errors, but I don't think it needs anything else tbh.

Reply:
sure thing, just tested and the errors are gone, so if it works without the need to manually specify a width and height then it's all good.